packages/material-ui/src/Select/SelectInput.js

```diff
@@ -25,18 +25,16 @@ const SelectRoot = experimentalStyled(
       const { styleProps } = props;
       return {
         [`&.${selectClasses.select}`]: {
-          // TODO v5: remove `root` and `selectMenu`
           ...styles.root,
```

Comment:
We should remove both `root` and `selectMenu`. The reason behind it is that this component does not have a root element (one that wraps all elements). We should drop it too from the `styleOverrides`, and state this as a breaking change.

Reply:
For consistency, we should rename the `SelectRoot` to `SelectSelect`, right? This applies to `NativeSelectInput` too.

Reply:
Let's do it 👍